src/app/retrospective/sprint-detail/sprint-detail.component.ts

```diff
@@ -248,4 +259,12 @@ export class SprintDetailComponent implements OnInit, OnDestroy  {
     toggleAutoRefresh() {
         this.enableRefresh = !this.enableRefresh;
     }
+
+    onChildRefreshStart($event) {
+        this.activeChildTabRefreshComplete = false;
+    }
+
+    onChildRefreshEnd($event) {
```

Comment:
Shouldn't we use a shared service to refresh the child component?
For example, the parent can call `this.refreshService.refreshChild$.next();` and disable its button.
The child tab can subscribe to this service to refresh its data.
And then, the child can call `this.refreshService.finishedChildRefresh$.next();`
The parent can subscribe to this and enable its button.

Reply:
Yeah, we can but a new issue should be created for this since it was missed in the refresh implementation itself.